docs/specification/README.md

```diff
@@ -156,7 +144,7 @@ A Chunk is a collection of compressed channel info and message records.
 
 #### Message Index (op=0x06)
 
-The Message Index record maps timestamps to message offsets. One message index record is written for each channel in the preceding chunk. All message index records for a chunk must immediately follow the chunk.
+The Message Index record maps timestamps to message offsets. If message indexing is in use, following each chunk, a message index record is written for each channel in the chunk preceding. All message index records for a chunk must immediately follow the chunk in a contiguous run of records.
```

Comment:
Did we decide whether MessageIndex is optional?

Reply:
yeah - implied I think by "If message indexing is in use"? What do you think?